css/print.css

```diff
@@ -1,3 +1,28 @@
-#menu-toggle {
-  display: none;
```

Comment:
Why get rid of this?

Reply:
that element wasn't present when using the old TOC

Reply:
It is not yet the case that "printing" is equivalent to "old TOC".

I guess it's fine landing now but I'd've preferred doing it as part of that.

Reply:
I guess the new TOC is hidden when printing anyway so it doesn't matter.